chpl/chpl-resources/src/main/resources-dev/jobs.xml

```diff
@@ -902,11 +902,11 @@
                 </entry>
             </job-data-map>
         </job>
-        <!--  Updates all g1g2 "ACI" MACRA measures to "PI" -->
+        <!-- Updates all MACRA measures to based on new requirements -->
```

Comment:
Suggest removing 'to', grammar. Remove in the description tag too, and everywhere else it was copied.